<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>getLineDialog</class>
 <widget class="QDialog" name="getLineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Створити файл</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background:white;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>271</width>
     <height>38</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>271</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color:white;
	margin-right:3%;
	margin-top:5px;
	border-width:1px;
	border-style:solid;
	border-color:rgb(204, 204, 204);
	border-radius:10px;
	padding: 5px 9px;
	font: italic 12pt &quot;Times New Roman&quot;;
	color:rgb(64, 65, 66);
}

QLineEdit:hover {
		border-bottom: 3px solid rgb(204, 204, 204);
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_setDataToCreateFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:white;
	margin-right:3%;
	margin-top:5px;
	border-width:1px;
	border-style:solid;
	border-color:rgb(204,204,204);
	border-radius:6px;
	padding: 5px 9px;
	font: italic 12pt &quot;Times New Roman&quot;;
	color:rgb(64, 65, 66);
image:url(:/photo/add_new_file.png);
image-position:left;
}

QPushButton:hover {
		border-bottom: 3px solid rgb(204, 204, 204);
}</string>
   </property>
   <property name="text">
    <string>    Створити</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>378</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	color:rgb(64,65,66);
	
	font: 12pt &quot;Times New Roman&quot;;
}</string>
   </property>
   <property name="text">
    <string>  Розширення файла буде додане за замовчуванням: &quot;.txt&quot;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>283</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton_clear">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	color:rgb(64,65,66);
	
	font: 10pt &quot;Times New Roman&quot;;
}</string>
      </property>
      <property name="text">
       <string>Очистити дані, які використовувались</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_keep">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	color:rgb(64,65,66);
	
	font: 10pt &quot;Times New Roman&quot;;
}</string>
      </property>
      <property name="text">
       <string>Залишити дані, які використовувались</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
